### ISSUE TITLE
#7274 [mB] replace ck editor - helptext for initiators is missing 

### ISSUE THREAD

**Issue by pascalkienast** [liqd/a4-meinberlin#5395]
**URL:** https://meinberlin-dev.liqd.net
**user:** initiators
**expected behaviour:** There should be an helptext for initiators pointing out they should enter alt text/ make alt text obligatory in the CKEditor
**behaviour:** The helptext is missing
**important screensize:** any
**device & browser:** any
**Comment/Question:** see story for the wording

Screenshot?
